eth/downloader/downloader.go

```diff
@@ -443,9 +384,6 @@ func (d *Downloader) syncWithPeer(p *peerConnection, hash common.Hash, td *big.I
 		// reset on error
 		if err != nil {
 			d.mux.Post(FailedEvent{err})
-		} else {
-			latest := d.lightchain.CurrentHeader()
```

Comment:
You probably still need to post an event when syncing is done, no? Just need to get `latest` from a different method